data_science_2/README.md

```diff
@@ -0,0 +1,59 @@
+### Overview
+
+Desde que pasó un asteroide multicolor extraño cerca de la tierra se están dando casos de una enfermedad extraña y letal llamada Fatal Idiopathic Clownitis, o FIC. Los síntomas clínicos del FIC son:
+
+* al auscultar el tórax se escucha [Killer Klowns from Outer Space](https://www.youtube.com/watch?v=tGVX033PiDA)
+* el paciente sufre lesiones cutáneas que recuerdan a un payaso
+* el paciente experimenta compulsiones cada vez más fuertes de hacer el payaso y cometer asesinatos en serie
+
+![killer_klown](killer_klown.png)
+
+Existe un fármaco experimental llamado E114 que consigue curar la enfermedad si se administra pronto: por desgracia, cuando hay síntomas clínicos ya es demasiado tarde y el fármaco ya no puede curar la enfermedad.
+
+Hay 2 compañías que han desarrollado tests para diagnóstico precoz y han llevado a cabo [trials randomizados](https://en.wikipedia.org/wiki/Randomized_controlled_trial) administrando el test y haciendo seguimiento clínico en muestras aleatorias de población:
+
+* La megapharma GSKPAZ ha desarrollado un test llamado `ultradetect` con una [accuracy (o exactitud)](https://en.wikipedia.org/wiki/Accuracy_and_precision#In_binary_classification) de *99.3%*, con datos del trial en [ultradetect_trial.csv](ultradetect_trial.csv)
+* Un equipo universitario infrafinanciado ha desarrollado un test llamado `test_b` con una accuracy de *92.1%*, con datos del trial en [test_b_trial.csv](test_b_trial.csv)
+
+Vistos los datos de accuracy en el "viaje de trabajo" a Hawaii financiado por GSKPAZ, el Ministerio de Sanidad está a punto de aprobar una campaña de screening masivo usando el test `ultradetect` pero a un funcionario de bajo nivel [le huele a chamusquina](https://en.wikipedia.org/wiki/Bad_Pharma) y te ha encargado que verifiques los cálculos y las conclusiones.
+
+Los datos de los trials vienen en csv, con las siguientes columnas:
+
+* `test_positive`: si el test de detección precoz ha dado un resultado positivo **1** o negativo **0**
+* `sick`: si el seguimiento clínico del paciente ha determinado que estaba enfermo de FIC **1** o no enfermo **0**
+
+#### ejercicio 0
+
+Como los trials han usado muestras poblacionales aleatorias podemos estimar [la prevalencia](https://es.wikipedia.org/wiki/Prevalencia) del FIC haciendo pooling de los datos de ambos trials.
```

Comment:
I've noticed that this is the only reference link in Spanish. Is it intentional?

I don't understand what "data pooling" means in this context. Does it mean combining the data from both trials?

Reply:
Nope, wasn't intentional, fixed.

I've clarified what's to be pooled & added link to wiki on pooling epidemiological data.